### PR TITLE
Image repeat

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review. Thanks!

### CODE REVIEW

File: web/scripts/editor/services/svc-background-parser.js [hunk 1]
```diff
@@ -78,7 +91,13 @@ angular.module('risevision.editor.services')
         if (background && background.useImage) {
           backgroundStyle += backgroundStyle ? ' ' : '';
           backgroundStyle += 'url(\'' + background.image.url +
-            '\') no-repeat';
+            '\')';
+
+          if (background.image.repeat) {
```
Comment (alex-deaconu) [inline web/scripts/editor/services/svc-background-parser.js]: @ezequielc I suggest you add `&& !background.image.scale` here to ensure the backgroundScaleToFit is not selected. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc one more suggestion here. Looks good otherwise, but I think this one's important. Thanks!

**Comment by ezequielc** [comment]
Good catch @alex-deaconu . Fixed. Please review. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc looks good. Apologies for the delay. Thanks!

**Comment by ezequielc** [comment]
No worries @alex-deaconu Thanks!

**Pull request merged**